cmd/promoted-image-governor/main.go

```diff
@@ -262,6 +272,47 @@ func isMirroredFromOCP(tag api.ImageStreamTagReference, refs []ImageStreamRef) b
 	return false
 }
 
+type githubClient interface {
+	GetRef(org, repo, ref string) (string, error)
```

Comment:
Use Git please, this will consume way too many tokens.

Reply:
Done.